docs/source/croissant.md

```diff
@@ -14,17 +14,16 @@ The [specification](https://github.com/mlcommons/croissant/blob/main/docs/croiss
 
 ## Get the metadata
 
-This guide shows you how to use Datasets Server's `/croissant` endpoint to retrieve the Croissant metadata associated to a dataset.
+This guide shows you how to use Hugging Face `/croissant` endpoint to retrieve the Croissant metadata associated to a dataset.
```

Comment:
I'll add it to the Hub's docs (https://github.com/huggingface/hub-docs/blob/main/docs/hub/api.md), so that we can link to it from here.

By the way, maybe all this page should go to https://github.com/huggingface/hub-docs, and we let /croissant undocumented here.

Reply:
OK, I created PRs for this:
- https://github.com/huggingface/datasets-server/pull/2643
- https://github.com/huggingface/hub-docs/pull/1259
- https://huggingface.co/spaces/enzostvs/hub-api-playground/discussions/3